### ISSUE TITLE
Task: recommended networking solution for 830 release  

### ISSUE THREAD

**Issue by XiaoningDing** [futurewei-cloud/arktos#567]
**What would you like to be added**:
provides tool and instructions for cluster admin to add more worker nodes to arktos cluster, particularly using a _proven-to-work_ cni plugin.

**Why is this needed**:
arktos-up script starts a single-noded cluster. At some cases, people may want to have a cluster having multiple worker nodes.

**Comment by h-w-chen** [comment]
will try below cni plugins
* flannel
* weave-net
* kube-router
* calico

stretch goal:
  able to run arktos having multiple worker nodes

**Comment by h-w-chen** [comment]
Blocked by flannel pod cannot be scheduled, which is regression introduced by arktos code (bug 588). fix PR #593 is under review.

**Comment by h-w-chen** [comment]
found another bug in kubelet: #595.

**Comment by h-w-chen** [comment]
PR #600 is under review; it provides fix to the bove issue #595.

**Comment by h-w-chen** [comment]
hit another issue #601 
it seems env setup issue - lack of some require parameters when cluster is started.

**Comment by h-w-chen** [comment]
PR #617 is under review. This PR makes the script tool hack/arktos-worker-up.sh able to add more worker nodes to arktos cluster.

**Comment by h-w-chen** [comment]
encountered by another issue #618. it seems the worker node unable to access kubernetes api server by the service ip 10.0.0.1.

**Comment by h-w-chen** [comment]
found another issue #618; we may need to create unique kubelet cert for each worker node.

**Comment by h-w-chen** [comment]
provide user instructions about how to workaround issue #618; the instruction is in PR #632.

**Comment by h-w-chen** [comment]
we had confirmed that Flannel works w/ multi-node arktos env;
we had instructions for admin to start up arktos master and to add more worker nodes;
we also verified that calico not working with arktos as-is.

Based on the task scope, this can be closed.
